drone/format/format.go

```diff
@@ -1,20 +1,13 @@
 package format
 
 import (
-	"bytes"
-	"io"
-	"io/ioutil"
-	"os"
-
-	"github.com/drone/drone-yaml/yaml"
-	"github.com/drone/drone-yaml/yaml/pretty"
 	"github.com/urfave/cli"
 )
 
 // Command exports the fmt command.
 var Command = cli.Command{
 	Name:      "fmt",
-	Usage:     "format the yaml file",
+	Usage:     "<deprecated. this operation is a no-op> format the yaml file",
```

Comment:
I think we can add `Hidden: true` as well

Reply:
done